core/templates/dev/head/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -9,13 +10,29 @@
   admin-navbar .oppia-navbar-profile-admin .open > a:hover,
   admin-navbar .oppia-navbar-profile-admin .open > a:focus,
   admin-navbar .oppia-navbar-tabs-admin > li > a:hover,
-  admin-navbar .oppia-navbar-tabs-admin > li > a.active {
+  admin-navbar .oppia-navbar-tabs-admin > li > a.active,
+  admin-navbar .oppia-admin-page-dropdown-icon {
     background-color: #fff;
     color: #00376d;
   }
   admin-navbar .oppia-navbar-admin {
     top: 0;
   }
+  @media screen and (max-width:710px){
+    .oppia-admin-page-dropdown{
+      display:block;
+    }
+    .tabs-to-hide{
```

Comment:
Please note the guidance on the coding style guide. Any class names defined by us should be prefixed with .oppia.

Alternatively, prefix them by admin-navbar or similar so that they apply only to this directive and do not end up applying by accident to other elements in different components.

Reply:
@seanlip , made some changes in new commit.